resources/views/admin/auth-clients/index.blade.php

```diff
@@ -53,6 +52,13 @@
 
         <div class="px-3 page-content">
             <div class="row">
+                <div class="col" align="left">
```

Comment:
Why are showing above the table only the first client with password grant enabled?
@velkymx @nolanpro should be better add a checkbox to enable the Password Grant type in the OAuth Client Screen, something like this:

![image](https://user-images.githubusercontent.com/8028650/61255385-57864280-a736-11e9-827c-686d6dbc408e.png)

similar to other OAuth applications:
![image](https://user-images.githubusercontent.com/8028650/61255402-6ec53000-a736-11e9-8992-95e47f713750.png)